MediaBrowser.MediaEncoding/BdInfo/BdInfoExaminer.cs

```diff
@@ -190,12 +190,12 @@ private void AddSubtitleStream(List<MediaStream> streams, TSTextStream textStrea
         private void AddSubtitleStream(List<MediaStream> streams, TSGraphicsStream textStream)
         {
             streams.Add(new MediaStream
-                {
-                    Language = textStream.LanguageCode,
-                    Codec = textStream.CodecShortName,
-                    Type = MediaStreamType.Subtitle,
-                    Index = streams.Count
-                });
+            {
+                Language = textStream.LanguageCode,
+                Codec = textStream.CodecShortName,
+                Type = MediaStreamType.Subtitle,
+                Index = streams.Count
+            });
```

Comment:
this block got me a bit confused since I expected it to be indented one level, would love clarification here

Reply:
Since it is part of a new object constructor as a parameter, the formatter puts it at the same level as the original statement.